shell/platform/linux/fl_touch_manager.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/fl_touch_manager.h"
+
+static const int kMinTouchDeviceId = 0;
+static const int kMaxTouchDeviceId = 128;
+
+struct _FlTouchManager {
+  GObject parent_instance;
+
+  GWeakRef view_delegate;
```

Comment:
This can be replaced with a reference to the engine - the view delegate only forwards the events to the engine. The existing delegates have been reduced or removed as part of the multi-view work.